dig.go

```diff
@@ -211,6 +217,20 @@ func New(opts ...Option) *Container {
 	return c
 }
 
+// DeferAcyclicVerification is an Option to override the default behavior
+// of container.Provide, disabling the validation the dependency graph remains
```

Comment:
nit: this comment needs updating. Otherwise looks good to me. I didn't thoroughly verify the correctness of your code, I'll leave that to the folks who actually own dig and know the internals better.

Reply:
I actually did rewrite this a bit. Is your objection to the word "disabling"? I think my sentence is confusing. This one combined with the last sentence in the comment block is intended to indicate the container will no longer validate on each Provide, but it will before the first Invoke.